python/test/unit/mesh/test_mesh.py

```diff
@@ -578,3 +579,16 @@ def test_original_index():
     s = sum(mesh.topology.original_cell_index)
     s = MPI.COMM_WORLD.allreduce(s, MPI.SUM)
     assert(s == nx**3 * 6 * (nx**3 * 6 - 1) // 2)
+
+
+def test_boundary_identification():
```

Comment:
Add a docstring - it's not clear from the function name what precisely it's testing.